### PR TITLE
Test/8685 implement missing integration tests

### PR DESCRIPTION
**Pull request by danielevalverde**

### CODE REVIEW

File: app/controllers/test_controller.rb [hunk 1]
```diff
@@ -118,6 +118,13 @@ def new_task
     render_success 'task', t
   end
 
+  def new_team_data_field()
```
Comment (caiosba) [inline app/controllers/test_controller.rb:121]: please remove the `()`

File: app/controllers/test_controller.rb [hunk 2]
```diff
@@ -118,6 +118,13 @@ def new_task
...
+    Team.current = Team.find(params[:team_id])
+    team_data_field = "Team-#{params[:fieldset]}"+Time.now.to_s
```
Comment (caiosba) [inline app/controllers/test_controller.rb:123]: Please add spaces around `+`

Comment (caiosba) [reply app/controllers/test_controller.rb:123]: Actually it's better to rewrite the line like this: `team_data_field = "Team-#{params[:fieldset]}-#{Time.now}"` - this way you avoid mixing interpolation and concatenation in the same line.

File: app/controllers/test_controller.rb [hunk 3]
```diff
@@ -118,6 +118,13 @@ def new_task
...
+    tt = create_team_task({label:team_data_field, team_id: params[:team_id], fieldset: params[:fieldset]}.merge(params))
```
Comment (caiosba) [inline app/controllers/test_controller.rb:124]: Please add a space after `label:`

File: app/controllers/test_controller.rb [hunk 4]
```diff
@@ -118,6 +118,13 @@ def new_task
...
+    render_success 'task', tt
```
Comment (caiosba) [inline app/controllers/test_controller.rb:125]: Since we're creating a `TeamTask` and not a `Task`, it's better to return `team_task` here, no?

**Review by caiosba — CHANGES_REQUESTED** [review]
Please check my comments inline.

**Review by melsawy — APPROVED** [review]

**Pull request merged**